### DISCUSSION TITLE
1.3.0 -> 1.3.1 significant size increase

### DISCUSSION THREAD

**Discussion by lazna** [gerardog/gsudo#227] [Q&A] [answered]
Using gsudo.exe in size matter redistributable package, and just notice the size of executable file increase like 30 times (!!!) between 1.3.0 and 1.3.1 version Why is this?

**Comment by gerardog** [comment, marked as answer]
Migration to .NET 7 is the cause.

Read this for details 
https://gerardog.github.io/gsudo/blog/gsudo-dotnet-7
